README.md

```diff
@@ -10,3 +10,38 @@ To generate a new HTML format locally run: ```bin/render_api```
 (requires [aglio](https://github.com/danielgtaylor/aglio))
 
 API docs accessible from /api.html
+
+## Local Setup
+
+To get the application running locally, you need to:
+
+ * ### Clone the repository
+ 	``` git clone git@github.com:ministryofjustice/defence-request-service-auth.git```
+
+ * ### Install ruby 2.2.2
+ 	It is recommended that you use a ruby version manager such as [rbenv](http://rbenv.org/) or [rvm](https://rvm.io/)
+
+ * ### Install postgres
+ 	http://www.postgresql.org/
+
+ * ### Bundle the gems
+       cd defence-request-service-auth
+       bundle install
+
+ * ### Create and migrate the database; run seeds
+
+ 		bundle exec rake db:create
+ 		bundle exec rake db:migrate
+ 		bundle exec rake db:seed
+
+
+ * ### Start the server
+ 		cd defence-request-service-auth && bundle exec rails server
+
+ * ### Use the app
+
+ 	You can find the service app running on `http://localhost:45454`
+
```

Comment:
Can we add a line about looking in the seeds file to find the login details of seeded users please?